src/core/thread/dua_manager.cpp

```diff
@@ -303,7 +303,18 @@ void DuaManager::UpdateCheckDelay(uint8_t aDelay)
 
 void DuaManager::HandleNotifierEvents(Events aEvents)
 {
-    Mle::MleRouter &mle = Get<Mle::MleRouter>();
+    Mle::MleRouter  &mle = Get<Mle::MleRouter>();
+    Lowpan::Context context;
```

Comment:
Can move `context` declaration to inside `if (aEvent.Contain(kEventThreadNetdataChanged))` block?

Reply:
Thanks! Resolved.